diff --git a/CMakeLists.txt b/CMakeLists.txt
index b5a9311..ec132fb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8.12)
 project(rabbitmq-c "C")
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
 
 # Enable MACOSX_RPATH by default. See: cmake --help-policy CMP0042
 if (POLICY CMP0042)
@@ -259,13 +261,13 @@ endif()
 
 
 option(BUILD_SHARED_LIBS "Build rabbitmq-c as a shared library" ON)
-option(BUILD_STATIC_LIBS "Build rabbitmq-c as a static library" ON)
+set(BUILD_STATIC_LIBS NOT ${BUILD_SHARED_LIBS})
 
-option(BUILD_EXAMPLES "Build Examples" ON)
-option(BUILD_TOOLS "Build Tools (requires POPT Library)" ${POPT_FOUND})
-option(BUILD_TOOLS_DOCS "Build man pages for Tools (requires xmlto)" ${DO_DOCS})
-option(BUILD_TESTS "Build tests (run tests with make test)" ON)
-option(BUILD_API_DOCS "Build Doxygen API docs" ${DOXYGEN_FOUND})
+option(BUILD_EXAMPLES "Build Examples" OFF)
+option(BUILD_TOOLS "Build Tools (requires POPT Library)" OFF)
+option(BUILD_TOOLS_DOCS "Build man pages for Tools (requires xmlto)" OFF)
+option(BUILD_TESTS "Build tests (run tests with make test)" OFF)
+option(BUILD_API_DOCS "Build Doxygen API docs" OFF)
 
 if (NOT BUILD_SHARED_LIBS AND NOT BUILD_STATIC_LIBS)
     message(FATAL_ERROR "One or both of BUILD_SHARED_LIBS or BUILD_STATIC_LIBS must be set to ON to build")
